### ISSUE TITLE
Failed to run on flutter 2.0.3

### ISSUE THREAD

**Issue by dangduoc** [znjameswu/flutter_math#46]
I got this error when trying to run flutter on andoird simulator.
 /flutter_math-0.2.1/lib/src/widgets/selection/overlay.dart:302:46: Too few positional arguments: 8 required, 7 given. child: selectionControls.buildToolbar ...

**Comment by expz** [comment]
The same happens for me with flutter 2.0.2 on the iphone simulator.

**Comment by baabale** [comment]
> I got this error when trying to run flutter on andoird simulator.
> /flutter_math-0.2.1/lib/src/widgets/selection/overlay.dart:302:46: Too few positional arguments: 8 required, 7 given. child: selectionControls.buildToolbar ...

I am also experiencing the same problem. **Please, give it a higher priority to debug it.** - Developer.

**Comment by KNuggies** [comment]
flutter_math has not been maintained. Someone already fixed this on the fork flutter_math_fork, and it's on pub.dev until this project is maintained again. https://pub.dev/packages/flutter_math_fork

**Comment by dangduoc** [comment]
> flutter_math has not been maintained. Someone already fixed this on the fork flutter_math_fork, and it's on pub.dev until this project is maintained again. https://pub.dev/packages/flutter_math_fork

Thank you, any idea whether this package is gonna be maintained again or not?

**Comment by baabale** [comment]
> flutter_math has not been maintained. Someone already fixed this on the fork flutter_math_fork, and it's on pub.dev until this project is maintained again. https://pub.dev/packages/flutter_math_fork

Awesome, thanks, buddy. It is working fine.

**Comment by rutvik110** [comment]
Yes,i too feaced the same issue today when building my old flutter web app.
The issue is in thie lib/src/widgets/selection/overlay.dart file whete the `selection.buildToolbar` is expecting 8 arguments instead of 7,
The last argument that's missiing is for the offset.So i solved this issue by passing the offset as the last argument there.
`child: selectionControls.buildToolbar(
          context,
          editingRegion,
          manager.preferredLineHeight,
          midpoint,
          [
            TextSelectionPoint(endpoint1, TextDirection.ltr),
            TextSelectionPoint(endpoint2, TextDirection.ltr),
          ],
          manager,
          clipboardStatus,
          midpoint,//This right here is the missing argument that was not passed before
        ),`
